guestmetric/guestmetric_linux.go

```diff
@@ -149,11 +149,31 @@ func (c *Collector) CollectNetworkAddr() (GuestMetric, error) {
 		}
 		paths = append(paths, prefixPaths...)
 	}
-
 	for _, path := range paths {
 		iface := filepath.Base(path)
 		vifId, err := getVifId(iface)
 		if err != nil {
+			macAddress, err := readSysfs(path+"/address")
```

Comment:
Can we please move this new section into a separate function?
Reasoning:
- It's not obvious to me what this code is there for. Needs a descriptive function name,
- It's nested too deep. That makes it hard to read.

Reply:
Sure. Will tweak it later on. Thank you very much.